src/animation/KeyframeTrack.js

```diff
@@ -190,25 +220,24 @@ THREE.KeyframeTrack.prototype = {
 	optimize: function() {
 
 		var newKeys = [];
-		var prevKey = this.keys[0];
+		var prevKey = this.keys[ 0 ];
 		newKeys.push( prevKey );
 
-		var equalsFunc = THREE.AnimationUtils.getEqualsFunc( prevKey.value );
+		for ( var i = 1; i < this.keys.length - 1; i ++ ) {
 
-		for( var i = 1; i < this.keys.length - 1; i ++ ) {
-			var currKey = this.keys[i];
-			var nextKey = this.keys[i+1];
+			var currKey = this.keys[ i ];
+			var nextKey = this.keys[ i +1 ];
```

Comment:
`var nextKey = this.keys[ i + 1 ];`

What happened? The beautifier should have corrected this pattern.